File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int Arrsum = 0;
        for(int x : nums){
            Arrsum += x;

        }
        int n = nums.size();

        int sum = (n * (n+1))>>1;

        return sum - Arrsum;

    }
};

File: 0268-missing-number/0268-missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n = nums.length ;
    let ExpSum = (n * (n+1)) / 2;
    let sum  = 0;
    for(let x of nums){
        sum += x;
    }

    return ExpSum - sum ;

};